### DISCUSSION TITLE
Add Bird's Nests to Item Identification.

### DISCUSSION THREAD

**Discussion by KevinCza** [runelite/runelite#16026] [Ideas]
At least empty and seed nests. They are impossible for me to tell apart.

**Comment by Enriath** [comment]
Currently, you can use `Inventory tags` to add a coloured outline to the empty nests, since those are all just one item ID (unlike seed nests). This same idea can be applied to other items that are too visually similar for you to tell apart at a glance.

**Comment by KevinCza** [comment]
I think part of the issue is I already have it outlined from a plugin that highlights based on price. I'll have to see if Inventory tags overrides. I didn't see anything when i last looked but thanks for the head's up.
